content/en/docs/reference/access-authn-authz/rbac.md

```diff
@@ -285,6 +285,29 @@ If you restrict `list` or `watch` by resourceName, clients must include a `metad
 For example, `kubectl get configmaps --field-selector=metadata.name=my-configmap`
 {{< /note >}}
 
+Rather than referring to individual `resources` and `verbs` you can use the wildcard `*` symbol to refer to all such objects.
+For `nonResourceURLs` you can use the wildcard `*` symbol as a suffix glob match and for `apiGroups` and `resourceNames` an empty set means that everything is allowed.
+Here is an example that allows access to perform any current and future action on all current and future resources (note, this is similar to the built-in `cluster-admin` role).
+
+```yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  namespace: default
+  name: very-open-role  # DO NOT USE THIS ROLE, IT IS JUST AN EXAMPLE
```

Comment:
with the change below, you could rename this something like `example.com-superuser`

Reply:
ok - done in https://github.com/kubernetes/website/pull/36043/commits/1532bf25437052f50f8cbee973f86cedd1d4ffc3